### ISSUE TITLE
Rubocop throws up a lot of offenses

### ISSUE THREAD

**Issue by harman28** [kenpratt/wikipedia-client#51]
I installed and ran [`rubocop`](https://github.com/bbatsov/rubocop) on the codebase:

```
> 19 files inspected, 351 offenses detected
```
The full results are [here](https://gist.github.com/harman28/b9a12f12038cb4f34d74fe2b9e504d66).

Many of these are style decisions rather than offenses, and can be filtered out by using the right preferences in a `.rubocop.yml` file. However, some are genuine issues, and can use some work.

Would be happy to take this up myself (if you see the merit in it), but I'll need occasional help and reviews from more experienced contributors whenever it comes a style decision.

**Comment by pietromenna** [comment]
@harman28: Is this solved by #54 ?

**Comment by harman28** [comment]
Yep!
```sh
wikipedia-client (master) > rake rubocop
Running RuboCop...
Inspecting 19 files
...................

19 files inspected, no offenses detected
```
